run_node.sh

```diff
@@ -2,8 +2,390 @@
 
 set -eo pipefail
 
+Help()
+{
+    echo "Run the aleph-node as either a validator or an archivist."
+    echo "Syntax: ./run_node.sh [--<name|image|stash_account> <value>] [--<archivist|mainnet|build_only|sync_from_genesis>] --ip|dns <value>" 
+    echo
+    echo "options:"
+    echo "archivist         Run the node as an archivist (the default is to run as a validator)"
+    echo "ip                The public IP of your node."
+    echo "dns               The public DNS of your node."
+    echo "stash_account     Stash account of your validator: optional but recommended, if you're re-running the script."
```

Comment:
At some point, we should deprecate `stash` notion everywhere, to follow fly-from-stash-accounts (https://forum.polkadot.network/t/staking-controller-deprecation-plan-staking-ui-leads-comms/2748). So this can be just named `validator_account` in the future. Now perhaps it's too early to change that as it will confuse our users

Reply:
Yeah, I think this is for the next PR but thanks for bringing this up!